ietf/static_src/css/iab.scss

```diff
@@ -177,3 +177,7 @@
 .iab-footer-image {
     height: 48px;
 }
+
+.body__iab {
+    background-color: rgba(62, 139, 193, 0.08);
+}
```

Comment:
This isn't an element of a `.body` block.  I think this should be something more like `.iab-body`.

FYI _if_ the body already had a `.body` class (which it does not), I'd suggest making it a modifier class`.body--iab`.

Reply:
All good, I'll make that change. I did some reading on BEM after our chat and figured that in this base the name would work out `body` being the element and `iab` being the modifier, but what I'm inferring is that BEM always wants a block to start with. I'll figure it out in time :-)